<ui version="4.0" >
 <class>ResilientPropagation</class>
 <widget class="QWidget" name="ResilientPropagation" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>128</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_17" >
     <property name="text" >
      <string>Maximum delta</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QLabel" name="label_16" >
     <property name="text" >
      <string>Minimum delta</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label_14" >
     <property name="text" >
      <string>Increase factor</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_15" >
     <property name="text" >
      <string>Decrease factor</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QDoubleSpinBox" name="increaseFactorSpin" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>238</red>
           <green>238</green>
           <blue>238</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>238</red>
           <green>238</green>
           <blue>238</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>238</red>
           <green>238</green>
           <blue>238</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Window" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>238</red>
           <green>238</green>
           <blue>238</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="readOnly" >
      <bool>false</bool>
     </property>
     <property name="decimals" >
      <number>1</number>
     </property>
     <property name="singleStep" >
      <double>0.100000000000000</double>
     </property>
     <property name="value" >
      <double>1.200000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QDoubleSpinBox" name="decreaseFactorSpin" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Dark" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>119</red>
           <green>119</green>
           <blue>119</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>238</red>
           <green>238</green>
           <blue>238</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="decimals" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <double>0.900000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.100000000000000</double>
     </property>
     <property name="value" >
      <double>0.500000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1" >
    <widget class="QDoubleSpinBox" name="minDeltaSpin" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>238</red>
           <green>238</green>
           <blue>238</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="decimals" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <double>5.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.100000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QDoubleSpinBox" name="maxDeltaSpin" >
     <property name="enabled" >
      <bool>true</bool>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Text" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>0</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Base" >
         <brush brushstyle="SolidPattern" >
          <color alpha="255" >
           <red>238</red>
           <green>238</green>
           <blue>238</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="decimals" >
      <number>1</number>
     </property>
     <property name="maximum" >
      <double>999.000000000000000</double>
     </property>
     <property name="singleStep" >
      <double>0.100000000000000</double>
     </property>
     <property name="value" >
      <double>50.000000000000000</double>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
